openmp/libomptarget/test/unified_shared_memory/shared_update.c

```diff
@@ -4,6 +4,8 @@
 
 // amdgpu runtime crash
```

Comment:
Fails with: "PluginInterface" error: Failure to synchronize stream (nil): Error in cuStreamSynchronize: an illegal memory access was encountered